impl/src/main/java/org/ehcache/config/builders/ExpiryPolicyBuilder.java

```diff
@@ -102,12 +134,20 @@ public ExpiryPolicyBuilder create(Duration create) {
    * @param access TTL since last access
    * @return this builder
    */
-  public ExpiryPolicyBuilder access(Duration access) {
+  public ExpiryPolicyBuilder<K, V> access(Duration access) {
     if (access != null && access.isNegative()) {
       throw new IllegalArgumentException("Access duration must be positive");
     }
-    this.access = access;
-    return this;
+    return access((a, b) -> access);
+  }
+
+  /**
+   * Set a function giving the TTL since last access
+   * @param access Function giving the TTL since last access
+   * @return this builder
+   */
+  public <K2 extends K, V2 extends V>ExpiryPolicyBuilder<K2, V2> access(BiFunction<K2, Supplier<? extends V2>, Duration> access) {
+    return new ExpiryPolicyBuilder<>(create, Objects.requireNonNull(access), update);
```

Comment:
Should it be legal to make multiple calls to the builder methods?

Reply:
Fair enough... might be good to document that the behavior is to over-write... someone might for example think they can specify multiple "access" rules and have the `earliest` expiration time apply.

Reply:
Documented